src/components/InlineCodeBlock/WrappedText.js

```diff
@@ -0,0 +1,87 @@
+import React, {Fragment} from 'react';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,'',Name,'','',is,'',Rajat],
```

Comment:
NAB might be good to update this because the result would be more like:

```suggestion
 *    [My,' ',Name,' ',' ',is,' ',Rajat],
```

Some quick local testing:

```
> let text = 'My name is Rory';
undefined
> text.split('\n').map(row => row.split(/(\s)/));
[ [
    'My',   ' ',
    'name', ' ',
    'is',   ' ',
    'Rory'
  ] ]
> text = 'My name  is  Rory';
'My name  is  Rory'
> text.split('\n').map(row => row.split(/(\s)/));
[
  [
    'My', ' ',    'name',
    ' ',  '',     ' ',
    'is', ' ',    '',
    ' ',  'Rory'
  ]
]
```

Reply:
Do we want to remove any empty strings from the matrix result here? Sorry, I'm just having some difficulty understanding how this component works.